src/components/composite/Product-cards/price-banner.js

```diff
@@ -0,0 +1,67 @@
+import { Button } from '../../atomic'
+
+export function PriceBanner({
+    totalPrice = 1000,
+    delivery = 10,
+    coupon = 20,
+    discount = 5,
+}) {
+    return (
+        <div className="eg-card card-one justify-space-around">
+            {totalPrice > 200 && (
+                <p>
+                    This is eligible for <span className="text-blue">Free</span>{' '}
+                    delivery.
+                </p>
+            )}
+            <header className="bold">
+                {' '}
+                Coupons
+                <Button
+                    btnType="secondary btn"
+                    btnText={
+                        <>
+                            <span className="material-icons outlined md">
+                                local_offer
+                            </span>{' '}
+                            Coupons
+                        </>
+                    }
+                />
+            </header>
+            <h2>Price Details:</h2>
+            <section>
+                <table>
```

Comment:
table can be a seperate component.

Reply:
Components should be reusable.
1. It is not used elsewhere.
2. It is not blocking the view as it's a behind-the-scenes component.